src/app/layouts/PageLayout/ReactivateAdsOverlay/index.tsx

```diff
@@ -97,8 +97,7 @@ export const ReactivateAdsOverlay = memo<Props>(({ onClose }) => {
               position={1}
               popup={popup}
               title="Act With Temple"
-              description={`Receive or send tokens, swap
-                and explore blockchain.`}
+              description={`Receive or send tokens, swap\nand explore blockchain.`}
```

Comment:
```suggestion
              description="Receive or send tokens, swap\nand explore blockchain."
```
Please remove template strings for other cases as well

Reply:
![Uploading Знімок екрана 2024-07-03 о 10.28.12.png…]()